shared/package.json

```diff
@@ -1,8 +1,18 @@
 {
   "name": "@aragon/templates-shared",
-  "version": "1.0.0",
-  "private": true,
+  "version": "1.0.0-rc.1",
   "description": "Shared helpers for Aragon templates",
+  "author": "Aragon Association <legal@aragon.org>",
+  "license": "GPL-3.0-or-later",
+  "files": [
+    "contracts/",
+    "ethereum-data",
+    "helpers/",
+    "lib/",
+    "scripts/",
+    ".env",
```

Comment:
I'm not really sure I understand the need for the `.env` file. I see we use `ETHDATA_LOCAL_PATH` in the docker-compose file, but does that get automatically injected by docker?

Reply:
We shouldn't add it here definitely. It is used by `dotenv` which we already dropped, so we don't need it here.
I don't see why adding `ethereum-data` and `docker-compose.yml`, these completely custom to our testing task.

Reply:
But we're sharing the testing utils with others; do you think it'd be useful if they had these geth-helpers too (like we do in our templates)?

Reply:
Now you're asking, yea, it's weird... maybe it is being injected by docker as you were saying